src/frontend/mosh-server.cc

```diff
@@ -336,6 +339,26 @@ int main( int argc, char *argv[] )
 static int run_server( const char *desired_ip, const char *desired_port,
 		       const string &command_path, char *command_argv[],
 		       const int colors, bool verbose, bool with_motd ) {
+  /* get network idle timeout */
+  long network_timeout = 0;
+  char *timeout_envar = getenv( "MOSH_SERVER_NETWORK_TMOUT" );
+  if ( timeout_envar ) {
+    /* We don't care about strtol errors here, they all produce sensible return values. */
+    network_timeout = strtol( timeout_envar, NULL, 10 );
+    if ( network_timeout < 0 ) {
+      network_timeout = 0;
```

Comment:
You could have mosh-server output a warning to mosh-client whenever the timeout is infinite. That would also cover the case where the user doesn't know that the default is infinite. (I'd still argue for a finite default, e.g. 30 days, but not sure if you guys are so keen on that idea.)